src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -90,20 +90,33 @@
     @RequestMapping(consumes = { MediaType.MULTIPART_FORM_DATA_VALUE }, method = POST)
     @ResponseStatus(HttpStatus.CREATED)
     public CatalogObjectMetadataList create(@PathVariable Long bucketId,
-            @ApiParam(value = "Name of the object") @RequestParam String name,
+            @ApiParam(value = "Name of the object ; if no name is provided, then the file should be a ZIP archive containing objects to create into the catalog") @RequestParam(required = false) Optional<String> name,
```

Comment:
Instead of should, I would use something else to better indicate failure in case no name is provided. 
Something like:

"Name of the object or empty when a ZIP archive is uploaded[. All objects inside the archive are stored inside the catalog.]"